### ISSUE TITLE
Missing Makevars

### ISSUE THREAD

**Issue by schloerke** [jimhester/covr#124]
Can the default for `set_makevars`'s `new_path` be file.path("~", ".R", "Makevars_covr_tmp"), or something that is findable.

Had a couple instances where my original file went missing as the `on.exit` code didn't get a chance to run.

-Barret

(Amazing package! Thank you!)

**Comment by jimhester** [comment]
CRAN packages are not supposed to write to anywhere but `tempdir()`. I could make the temporary filename based on the actual filename, but I cannot put in in `~/.R`.

Your original file should still be in your OSs temporary directory unless you have restarted the computer or cleaned out the directory.

`on.exit()` is supposed to run even in the case of error from the function, so it is surprising that you ran into this situation.

Anyway I will make a note about improving the temporary filename.

**Comment by schloerke** [comment]
Ok. Good to know about the files!

Situation that happened... 
* My Makevars file didn't exist before.  
* The temp lines for `CFLAGS`, `CXXFLAGS`, and `FFLAGS` were added to my ~/.R/Makevars
* (While specific,) With a new Makevars file, I couldn't install stringi at a later time.
* I did clean out my temp files, but not while R was running.

Looking at the covr code again, I believe my ~/.R/Makevars situation is a freak accident.

Thanks again!
